server/Customer.java

```diff
@@ -59,4 +61,17 @@ public String getUsername() {
 	public String getPassword() {
 		return password;
 	}
+
+	public void setAccountLocked(){
+		this.accountLocked = true;
+	}
+
+	public void setUnlockAccount(){
```

Comment:
`setAccountLocked()` and `setUnlockAccount()` could have been just one function, something like:

```java
public void setAccountLocked(boolean value) {
  accountLocked = value;
}
```

I'm not sure which way I like more, just dropping ideas :)